docs/teamgrid.rst

```diff
@@ -0,0 +1,37 @@
+..
```

Comment:
Don't commit this file. Committing autogenerated files just causes problems later.